packages/rules-unit-testing/src/api/index.ts

```diff
@@ -355,64 +352,98 @@ export function loadFirestoreRules(
     throw new Error('must provide rules to loadFirestoreRules');
   }
 
-  return new Promise((resolve, reject) => {
-    request.put(
-      {
-        uri: `http://${getFirestoreHost()}/emulator/v1/projects/${
-          options.projectId
-        }:securityRules`,
-        body: JSON.stringify({
-          rules: {
-            files: [{ content: options.rules }]
-          }
-        })
-      },
-      (err, resp, body) => {
-        if (err) {
-          reject(err);
-        } else if (resp.statusCode !== 200) {
-          console.log('body', body);
-          reject(JSON.parse(body).error);
-        } else {
-          resolve();
-        }
+  const resp = await requestPromise(request.put, {
+    method: 'PUT',
+    uri: `http://${getFirestoreHost()}/emulator/v1/projects/${
+      options.projectId
+    }:securityRules`,
+    body: JSON.stringify({
+      rules: {
+        files: [{ content: options.rules }]
       }
-    );
+    })
   });
+
+  if (resp.statusCode !== 200) {
+    throw new Error(JSON.parse(resp.body.error));
+  }
 }
 
 export type ClearFirestoreDataOptions = {
   projectId: string;
 };
-export function clearFirestoreData(
+export async function clearFirestoreData(
   options: ClearFirestoreDataOptions
 ): Promise<void> {
   if (!options.projectId) {
     throw new Error('projectId not specified');
   }
 
-  return new Promise((resolve, reject) => {
-    request.delete(
-      {
-        uri: `http://${getFirestoreHost()}/emulator/v1/projects/${
-          options.projectId
-        }/databases/(default)/documents`,
-        body: JSON.stringify({
-          database: `projects/${options.projectId}/databases/(default)`
-        })
-      },
-      (err, resp, body) => {
-        if (err) {
-          reject(err);
-        } else if (resp.statusCode !== 200) {
-          console.log('body', body);
-          reject(JSON.parse(body).error);
-        } else {
-          resolve();
-        }
-      }
+  const resp = await requestPromise(request.delete, {
+    method: 'DELETE',
+    uri: `http://${getFirestoreHost()}/emulator/v1/projects/${
+      options.projectId
+    }/databases/(default)/documents`,
+    body: JSON.stringify({
+      database: `projects/${options.projectId}/databases/(default)`
+    })
+  });
+
+  if (resp.statusCode !== 200) {
+    throw new Error(JSON.parse(resp.body.error));
+  }
+}
+
+/**
+ * Run a setup function with background Cloud Functions triggers disabled. This can be used to
+ * import data into the Realtime Database or Cloud Firestore emulator without triggering locally
+ * emulated Cloud Functions.
+ *
+ * This method only works with Firebase CLI version 8.13.0 or higher.
```

Comment:
Do you mind adding this comment to changelog to provide some details?

Reply:
Done, although I wasn't quite sure about the changelog format.  Most of the other messages seem to be machine generated?

Reply:
Sorry, I wasn't clear. I meant adding it to the changeset. Yes, the changelog is generated from changesets.

Reply:
Got it, fixed!